data/scenarios/Challenges/telephone.yaml

```diff
@@ -0,0 +1,223 @@
+        (if (hash % 7 <= 1) then {X} else if (hash % 7 <= 3) then {O} else {grass})
+        else {grass}
+    ]
+  upperleft: [0, 0]
+  offset: false
+  palette:
+    '.': [grass]
+    '#': [grass, wall]
+    '%': [grass, blueborder]
+    'A': [stone, mountain]
+    'T': [grass, tree]
+    'W': [stone, water]
+    '~': [stone, wavy water]
+    '^': [grass, null, base]
+    '&': [grass, purpleborder]
+    '=': [grass, track]
+    'S': [grass, track, shuttle]
+    'P': [grass, null, photocopier]
+    'J': [grass, null, judge]
+    't': [terminal]
+  map: |
+    ########################################################################################################################
+    #..........................................................AA..........................................................#
+    #.........................................................AAA..................................................T.......#
+    #........................................................AAA..................................................TT.......#
+    #.................%%%%%%%%%%..............................AAAA..............................%%%%%%%%%%.........TTT.....#
+    #.................%........%...............................AAAAA............................%........%.......TTT.......#
+    #.................%........%..................................AA............................%........%.....TTTT........#
+    #.................%........%..............................AAAAA.............................%........%.......TTT.......#
+    #.................%J.......%.............................AAA................................%........%.........T.......#
+    #.................%%%%%%%%%%..&&&.......................AAAAA..........................&&&..%%%%%%%%%%.......TTTT......#
+    #.............................&P==========================AA====================S=======t&..............^.....TTTT.....#
+    #.............................&&&........................AAAA..........................&&&.....................TT......#
+    #..........................................................AAA.........................................................#
+    #...........................................................AAAA.........................................WWW...........#
+    #..........................................................AAAAAA........................................WW~W..........#
+    #...........................................................AAAA.......................................WW~WW...........#
+    #..........................................................AAAA..........................................WW............#
+    #.........................................................AAA..........................................................#
+    ########################################################################################################################
```

Comment:
While I was playing, I kind of wished that I could see the effect in on the left side without having to manually scroll the viewport.  What do you think about reducing the width of the map?